message/html/compose.js

```diff
@@ -178,7 +198,10 @@ exports.create = function (api) {
       function done (err, msg) {
         publishBtn.disabled = false
         if (err) throw err
-        else if (msg) textArea.value = ''
+        else if (msg) {
+          textArea.value = ''
+          api.drafts.sync.set(draftLocation, '')
```

Comment:
Need to check your module. I think we actuallt want to delete the draft. If this is the way you trigger that using your set api, I'd be suprised.

I would expect `api.drafts.sync.set(draftLocation, null)` would actually hard wipe the key in the store.
This is just knit-picking, but will help us from having a million keys with no content in them any more

Reply:
🔥 this is true! empty string isn't special cased. I did not forsee the acumulation of superseeded drafts just wanted to blank it out and yes, it would get messy in the long run and also not work for a `/drafts` page.

special casing `null` would also be weird though, i think and adding a `.remove(key)` function totally sounds reasonable to me.